shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -508,30 +509,50 @@ private int guessBottomKeyboardInset(WindowInsets insets) {
   public final WindowInsets onApplyWindowInsets(@NonNull WindowInsets insets) {
     WindowInsets newInsets = super.onApplyWindowInsets(insets);
 
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+      Insets systemGestureInsets = insets.getSystemGestureInsets();
+      viewportMetrics.systemGestureInsetTop = systemGestureInsets.top;
```

Comment:
Super tangential, just realized we added this to mediaquery. We should move the iOS home bar to use this too :)

Reply:
sorry disregard. After reading this from media query and android, I realized it doesn't mean the same thing